test/go.mod

```diff
@@ -7,8 +7,8 @@ replace sigs.k8s.io/cluster-api => ../
 require (
 	github.com/blang/semver v3.5.1+incompatible
 	github.com/go-logr/logr v0.4.0
-	github.com/onsi/ginkgo v1.16.3
-	github.com/onsi/gomega v1.12.0
+	github.com/onsi/ginkgo v1.16.4
+	github.com/onsi/gomega v1.13.0
```

Comment:
FYI, these come from controller-runtime. We should update them first?

Reply:
https://github.com/kubernetes-sigs/controller-runtime/pull/1550